play-routes-compiler/src/main/scala/CommandLinePlayRoutesCompiler.scala

```diff
@@ -91,6 +92,7 @@ object CommandLinePlayRoutesCompiler {
           config.additionalImports,
           config.generateForwardsRouter,
           config.generateReverseRouter,
+          config.jsReverseRouter,
```

Comment:
We use the javascript reverse routes quite a bit so we'll either want to just use `generateReverseRouter` for this rather than adding a new field, or make this configurable based on the command line options and add it to the bazel rules [here](https://github.com/lucidsoftware/rules_play_routes/blob/master/play-routes/play-routes.bzl#L87). Adding a command line option and an input to the bazel rule would be best, but a lot more work, so I think it's fine if you just want to use `config.generateReverseRouter` here and not bother with the other changes